harvester_e2e_tests/utils.py

```diff
@@ -336,14 +336,17 @@ def _wait_for_image_become_active():
 def create_vm(request, admin_session, image, harvester_api_endpoints,
               template='basic_vm', keypair=None, volume=None, network=None,
               cpu=1, disk_size_gb=10, memory_gb=1, network_data=None,
-              user_data=None, running=True):
+              user_data=None, running=True, volume2=None):
```

Comment:
For future consideration, I would suggest doing `additional_volumes=[]` instead of `volume2=None` in case we more volumes to add. We don't have to change it now, just something to keep in mind.